src/app/drive/views/RequestAccess/RequestAccess.tsx

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from 'app/store';
+import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
+import { LockSimple, CheckCircle } from '@phosphor-icons/react';
+import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+import Button from 'app/shared/components/Button/Button';
+import bigLogo from 'assets/icons/big-logo.svg';
+
+function RequestAccess(): JSX.Element {
+  const user = useSelector<RootState, UserSettings | undefined>((state) => state.user.user);
+  const { translate } = useTranslationContext();
+
+  const [messageText, setMessageText] = useState<string>('');
+  const [messageTextLimit, setMessageTextLimit] = useState<boolean>(false);
+  const [requestSent, setRequestSent] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (messageText.length >= 950) {
```

Comment:
Avoid use magic numbers. Add it to a const in order to know what represents the number or what is its purpose with the name

Reply:
Done